fiddler/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def dd_environment(instance):
+    yield instance
+
+
+@pytest.fixture(scope='session')
+def instance():
+    return {"url": "https://demo.trial.fiddler.ai", "fiddler_api_key": "apikey", "organization": "demo"}
```

Comment:
This file initializes the ddev E2E agent docker container with the sample config supplied by `def instance()`. 

To run the ddev E2E agent container, make sure docker-desktop is running and then run these commands inside the root of your development `integrations-extras` repository:

```
ddev -x env ls fiddler
ddev -x env start fiddler py38 --dev
```
Edit config file: `ddev -x env edit fiddler py38`

When the environment is successfully running, you can modify the E2E fiddler config file and input your config values. You'll no longer need to hardcode the config values in the code.

Docs for ddev E2E usage: https://datadoghq.dev/integrations-core/e2e/